docs/GettingStartedWindows.md

```diff
@@ -39,9 +39,21 @@
 ## Installation
 
 1. Install Visual Studio from [Microsoft](https://visualstudio.microsoft.com).
-2. Install Swift Toolchain from [//swift/build](https://compnerd.visualstudio.com/compnerd/swift-build).
+2. Install Swift Toolchain from either the GitHub [releases](https://github.com/compnerd/swift-build/releases) or the [//swift/build](https://compnerd.visualstudio.com/compnerd/swift-build) CI Pipeline.
 3. Deploy Windows SDK updates.  This must be run from an (elevated) "Administrator" `x64 Native Tools for VS2019 Command Prompt` shell.
 
+Before executing the following, you'll need to ensure that you've set the following values for the build parameter variables: 
+
+* **SystemDrive** 
```

Comment:
This should be set already by the system.  Could you check by running set in cmd?